pages/landing-page.js

```diff
@@ -0,0 +1,437 @@
+          height: 600px;
+          overflow: scroll;
+          margin: 20px;
+          border-radius: 10px;
+          box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
+        }
+        .section-title {
+          color: #0d2529b4;
+          font-size: 2.5rem;
+          font-weight: 500;
+          padding-bottom: 200px;
+        }
+
+        @media (max-width: 768px) {
+          .testimonial-section {
+            height: auto;
+            padding: 100px 0;
+
+            flex-direction: column;
+            align-items: center;
+            justify-content: center;
+          }
+          .cards {
+            flex-direction: column;
+            height: auto;
+            padding: 0;
+          }
+          .testimonial-card {
+            width: 100%;
+            height: 500px;
+            margin: 0;
+            border-radius: 0;
+            box-shadow: none;
+          }
+          .section-title {
+            font-size: 1.5rem;
+            padding-bottom: 50px;
+          }
+        }
+      `}</style>
+    </section>
+  );
+};
+
+// -----------------------------------------------------------------------------------------------------------------------
+// Export TestimonialSection end.
+// -----------------------------------------------------------------------------------------------------------------------
+
+// -----------------------------------------------------------------------------------------------------------------------
+// Main Component for this page lives here.
+// -----------------------------------------------------------------------------------------------------------------------
+
+const LandingPage = () => {
+  const router = useRouter();
+  return (
+    <>
+      <Head>
+        <title>{'Help a Family in Need'}</title>
+        <meta content={'Help a Family in Need'} name="description" />
+        <link href="https://www.margaritahumanitarian.org/" rel="canonical" />
+        <link href="/favicon.ico" rel="icon" />
+      </Head>
+      <LandingPageNav fadeInSpeed="1.5s" />
+      <main className="body">
+        <LargeHeroSection
+          bgImage="/images/HotMealDay.jpg"
+          heroHeight="85vh"
+          opacity="20"
+        >
+          <LargeHeroContent
+            fadeInSpeed="0.8s"
+            mainTextSize="lg"
+            title="Feed A Family Today"
+            titleSize="5xl"
+          >
+            <p className="mb-5">
+              {
+                'Families are in need more than ever. The pandemic coupled with trying economic times has really put the less fortunate in a difficult position as they try to maintain a functioning household. Give what you can today to help raise up those in need.'
+              }
+            </p>
+
+            <button
+              className="btn btn-accent rounded-full shadow-lg"
+              onClick={() => router.push('/in-kind')}
+              type="button"
+            >
+              {'Help Today'}
+            </button>
```

Comment:
Perhaps instead of a `button`, a `Link` from NextJS can be used instead.

Reply:
That's a good idea. @LowLifeArcade check out the Next.js `Link` docs at https://nextjs.org/docs/api-reference/next/link and let us know if you need any help figuring it out 😄